dev/setup_darwin.go

```diff
@@ -183,16 +193,7 @@ func InstallIntoSystem(listenPort, tlsPort int, dir, domains, timeout string) er
 		return errors.Context(err, "loading plist into launchctl")
 	}
 
-	fmt.Printf("* Installed puma-dev on ports: http %d, https %d\n", listenPort, tlsPort)
-
-	return nil
-}
-
-func Stop() error {
```

Comment:
moved this to `stop.go`, since it's used in both linux and darwin (and doesn't have anything to do with setup)